setup.py

```diff
@@ -64,8 +64,8 @@
         'SpiNNFrontEndCommon >= 1!4.0.1, < 1!5.0.0',
         'sPyNNaker >= 1!4.0.1, < 1!5.0.0',
         'quantities >= 0.12.1',
-        'pynn >= 0.8, < 0.9',
+        'pynn >= 0.9.1',
```

Comment:
Whilst I know that this is true, would it be better to say:
'pynn >= 0.8, < 1.0'

and then also:
'neo >= 0.3, < 0.6'

This would then allow the user to install pyNN 8 and have it work if they really wanted to - or does this change not allow pyNN 8 at all?

Reply:
Upper limits are probably a good idea and will be added.

As for pynn0.8 and neo 0.3 or neo 0.4 I am more reluctant for two reasons.

1. They only work if BOTH pynn and neo are a lower version

2. Andrew Davison has asked us not to use pynn0.8

Reply:
Upper limited added but NOT allowing 0.8 for reason above and because I would suggest announcing that support for that is deprecated